devtools-ci-index.yaml

```diff
@@ -1658,6 +1658,96 @@
            allow-empty: true
            fingerprint: true
 
+- job-template:
+    name: 'devtools-test-booster-{test_url}-{test_suite}-{quickstart_name}-{cluster}'
+    defaults: global
+    node: devtools
+    after: fabric8io-fabric8-test-build-master
+    gh_username: osiotestmachine
+    properties:
+        - github:
+            url: https://github.com/fabric8io/fabric8-test
+    scm:
+        - git:
+            url: https://github.com/fabric8io/fabric8-test.git
+
+            shallow_clone: true
+            branches:
+                - master
+    triggers:
+        - timed: '{ee_test_start_time}'
+        - build-result:
+            cron: '* * * * *'
```

Comment:
do we really want this to run every minute ?

Reply:
That cron setting is in the template (its actually in several templates - most notably the EE tests and a Che test). The individual job definitions each have explicit cron settings. 

But - still - this does look alarming. Can we simply remove these statements?

Reply:
I believe that's correct, yes